cli/internal/cache/cache_signature_authentication.go

```diff
@@ -44,21 +47,12 @@ func (asa *ArtifactSignatureAuthentication) generateTag(hash string, artifactBod
 
 func (asa *ArtifactSignatureAuthentication) getTagGenerator(hash string) (hash.Hash, error) {
 	teamId := asa.teamId
-	secret, err := asa.secretKey()
-	if err != nil {
-		return nil, err
-	}
-	artifactMetadata := &struct {
-		Hash   string `json:"hash"`
-		TeamId string `json:"teamId"`
-	}{
-		Hash:   hash,
-		TeamId: teamId,
-	}
-	metadata, err := json.Marshal(artifactMetadata)
+	secret, err := asa.getSecretKey()
 	if err != nil {
 		return nil, err
 	}
+	metadata := []byte(hash)
```

Comment:
Are there any cases where a plain byte slice could be different in Rust and Go?

Reply:
As long as the encodings match for the strings, I believe they should be identical.